### DISCUSSION TITLE
Inverse kinematic result is not limit checked

### DISCUSSION THREAD

**Discussion by yotabits** [tesseract-robotics/tesseract#662] [General]
https://github.com/ros-industrial-consortium/tesseract/blob/f8cb6ac56c33a9910077607de4aed898993ba7d4/tesseract_kinematics/src/opw/opw_inv_kin.cpp#L50-L83

It seems that the solutions that are returned by this method are not joint limit checked.
If my assumption is correct is this a deliberate design choice?
Thank you :)

**Comment by Levi-Armstrong** [comment]
It was, in the case of forward kinematics being used for calculating analytical jacobian introducing the limits cause issues when near the limits.
